test/e2e/e2e.go

```diff
@@ -140,7 +140,7 @@ func waitForActivatorEndpoints(resources *v1test.ResourceObjects, clients *test.
 
 		// The subset is set. But in theory it might be revision pods,
 		// so verify below that at least 1 address is an activator pod.
-		if presources.ReadyAddressCount(svcEps) != int(sks.Spec.NumActivators) {
+		if presources.ReadyAddressCount(svcEps) > int(sks.Spec.NumActivators) {
```

Comment:
We know how many activators are in the system via the first query here. Should we change this comparison to be

```golang
readyActivators := presources.ReadyAddressCount(aeps)
expectedActivators := min(readyActivators, sks.Spec.NumActivators)
if presources.ReadyAddressCount(svcEps) != expectedActivators
```

To make it bulletproof and actually meaningful? The "less than" check feels a little... weak.

Reply:
Thank you. Yes, I think it is the better condition. Updated.